src/content/advanced/access-management/iam-roles-for-service-accounts/index.md

```diff
@@ -495,6 +495,15 @@ Modify the trust entity of your AWS IAM roles with the new identity provider URL
 
 For cross account roles please follow the [guide](#cross-account) above and replace the `CLOUDFRONT_DOMAIN` with the new `CLOUDFRONT_ALTERNATE_DOMAIN`.
 
+Login into the AWS on the account where the cluster is running:
```

Comment:
This paragraph doesn't have an explanation. What is done here and why? Should this have a separate heading? Is it about making it work across accounts?

Reply:
It is actually part of cross account roles mentioned above:
> For cross account roles please follow the [guide](#cross-account) above and replace the `CLOUDFRONT_DOMAIN` with the new `CLOUDFRONT_ALTERNATE_DOMAIN`.

Maybe, we should go with adding a separate heading.

Reply:
I have updated the doc so, it doesn't relink to previous guide which might be confusing. Also, added screenshot.